src/datachain/lib/convert/sql_to_python.py

```diff
@@ -4,15 +4,11 @@
 from sqlalchemy import ColumnElement
 
 
-def sql_to_python(args_map: dict[str, ColumnElement]) -> dict[str, Any]:
-    res = {}
-    for name, sql_exp in args_map.items():
-        try:
-            type_ = sql_exp.type.python_type
-            if type_ == Decimal:
-                type_ = float
-        except NotImplementedError:
-            type_ = str
-        res[name] = type_
-
-    return res
+def sql_to_python(sql_exp: ColumnElement) -> Any:
```

Comment:
This function was updated to be consistent with [python_to_sql](https://github.com/iterative/datachain/blob/main/src/datachain/lib/convert/python_to_sql.py#L37) function (both take single element). Also it was used [in one place only](https://github.com/iterative/datachain/blob/main/src/datachain/lib/signal_schema.py#L493)  and for single element, no need to have `dict` as input param and output result. Also it is now used to detect `partition_by` type in `group_by` in case function is used.